app.R

```diff
@@ -0,0 +1,1267 @@
+              ),
+            
+               ##### Table and Plot on the right ----
+                  # h3("Table and Plot"),
+                  column(
+                    width = 8,
+                         align = "center",
+                         conditionalPanel(
+                           condition = "input.random == 'Null with equal probabilities'",
+                            tableOutput("equalValues"),
+                           
+                           conditionalPanel(
+                             condition = "input.random == 'Null with equal probabilities'",
+                             plotOutput(outputId = "equalPlot",
+                                        width = '90%',
+                                        click = "plot_click"),
+                             
+                             tableOutput("equalPlotClickedpoints"),
+  
+                             htmlOutput(
+                               outputId = "equalText",
+                               class = "text-center")
+                                          ))),
+
+                  column(
+                    width = 8,
+                         align = "center",
+                    
+                         conditionalPanel(
+                           condition = "input.random == 'Null with different probabilities'",
+                           tableOutput("diffValues"),
+                           
+                           plotOutput(outputId = "diffPlot",
+                                      width = "90%",
+                                      click = "plot_click"),
+                           
+                           tableOutput("diffPlotClickedpoints"),
+                           
+                            htmlOutput(
+                              outputId = "diffText",
+                              class = "text-center")
+                           )))
+        ),
+
+        #### Set up the Explore Page ----
+        tabItem(
+          tabName = "explore",
+          withMathJax(),
+          # Use h2 here
+          h2("Explore Your Own Data"),
+          p("You can use this page to see the behavior of distribution of the chi-square statistic for your data.
+                  If you need guidence for each part, put arrow on that box.
+                  When you finished, click on Simulate Now to see the plots."),
+          fluidRow(
+            #column of length 12 which is the whole width
+            #I include everthing in a column though because this way there are margins and it looks better
+            ##### sideBar on the left ----
+            column(
+                   width = 4,
+                   wellPanel(
+                     align = "left",
+                   textInput(inputId = "names",
+                             label = HTML("Level names", 
+                                          as.character(
+                                            actionLink(inputId = "namesPopover", 
```

Comment:
I think instead of having these as info buttons, these could just be included in the instructions in a bullet list